.pullapprove.yml

```diff
@@ -2,10 +2,10 @@ version: 3
 
 # see https://docs.pullapprove.com/config/groups/
 groups:
-  documentation: documentation changes require 2 approvals
-    description:
+  documentation:
+    description: documentation changes require 2 approvalss
```

Comment:
One too many 's'es at the end of "approvalss"

Reply:
Thanks, I fixed it.